src/features/inventory/index.js

```diff
@@ -13,6 +13,9 @@ class Inventory extends Component {
     this.state = {
       newItemIndicator: false
     };
+
+    this._toggleInventory = this._toggleInventory.bind(this);
+    this.handleKeyDown = this.handleKeyDown.bind(this);
```

Comment:
This is a bit confusing. Why are we doing something like this? Is it so that we wouldn't have to bind on lines 72 and 73?

If so, we can probably remove these two lines (17 and 18) altogether, and instead update 72 and 73 to look like this:
```
onClick={() => { this._toggleInventory(); }}
onKeyDown={() => { this.handleKeyDown(); }}
```

Reply:
Right now, I only see two options:

1. remove `no-invalid-this: warn` in eslintrc.yml.
2. use `react/jsx-no-bind` (avoid adding more dependencies to package.json) and no more `onClick={() => {}}` everywhere, up to and including stateless functional components. (:

Reply:
I'm in favor of option 1.

Reply:
Option 1 is the move, you can go ahead and remove that linter rule.

Reply:
Okay, @ASteinheiser @Alaricus, it's done. Two new commits up. I believe that's all for now.